src/build/partition.jl

```diff
@@ -101,13 +129,19 @@ function _partition_io!(d::Dict, set::Dict)
     d[:id0] = filter_with(d[:use], set)
     d[:ys0] = filter_with(d[:supply], set)
 
+    # In sectordisagg, the good/sector column names are switched...
+    if d[:sector]==:detail
+        x = Rename.([:g,:s,:g_temp],[:g_temp,:g,:s])
+        d[:ys0] = edit_with(d[:ys0], x)
+    end
```

Comment:
**For future**. When partitioning, `ys0`, WiNDC uses a different column order when defining detail-level supply/use. These should *definitely* be consistent.

Reply:
This is interesting. 
ir_supply and jc_supply are different. 
j and i are aliases and subsets of ir_supply and jc_supply (pretty sure).
Something is wrong here - we should discuss this further and test a couple of things out.
I believe the first is correct.
The make tables are industry (rows) x commodity (columns).
In ys0(r,g,s) --- g corresponds to commodity and s to industry (basically).
So I think partitionbea is correct.

Reply:
This could be why there seemed to be issues reconciling with BEA tables. If so, kind of an inconspicuous error.

Reply:
Nice catch!!!